style.css

```diff
@@ -1,97 +1,113 @@
-    font-size: 35px;
-    text-transform: uppercase;
-    font-weight: 800;
-    letter-spacing: 2px;
-    line-height: 60px;
-    margin-bottom: 30px;
+
+.content {
+  max-width: 650px;
+  margin: 60px 100px;
 }
-.content .info h2 span{
-    color: cornsilk;
-    font-size: 35px;
-    font-weight: 700;
+
+.content .info h2 {
+  color: crimson;
+  font-size: 35px;
+  text-transform: uppercase;
+  font-weight: 800;
+  letter-spacing: 2px;
+  line-height: 60px;
+  margin-bottom: 30px;
 }
-.content .info p{
-    font-size: 20px;
-    font-weight: 800;
-    margin-bottom: 40px;
+
+.content .info h2 span {
+  color: cornsilk;
+  font-size: 35px;
+  font-weight: 700;
 }
-header{
-    color: #000;
-    position: relative;
-    top: 0;
-    width: 100%;
-    padding: 30px 100px;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
+
+.content .info p {
+  font-size: 20px;
+  font-weight: 800;
+  margin-bottom: 40px;
 }
 
-header .logo{
-    position: relative;
-    color: crimson;
-    font-size: 15px;
-    text-decoration: none;
-    text-transform: uppercase;
-    font-weight: 800;
-    letter-spacing: 1px;
-    background: url(LYS01.jpg)no-repeat;
+header {
+  color: #000;
+  position: relative;
+  top: 0;
+  width: 100%;
+  padding: 30px 100px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
 
+header .logo {
+  position: relative;
+  color: crimson;
+  font-size: 15px;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: 800;
+  letter-spacing: 1px;
+  background: url(LYS01.jpg) no-repeat;
 }
-header .navigation a{
-    color: crimson;
-    text-decoration: none;
-    font-weight: 700;
-    letter-spacing: 1px;
-    padding: 2px 15px;
-    border-radius: 20px;
-    transition: 0.3s;   
-    transition-property: background; 
+
+.media-icons a {
+  position: relative;
+  color: #111;
+  font-size: 30px;
 }
-.header .navigation a:not(:last-child){
-    margin-right: 30px;
+
+header .navigation a {
+  color: crimson;
+  text-decoration: none;
+  font-weight: 700;
+  letter-spacing: 1px;
+  padding: 2px 15px;
+  border-radius: 20px;
+  transition: 0.3s;
+  transition-property: background;
 }
-.media-icons{
-    display: inline-flex;
-    justify-content: center;
-    align-items: center;
-    margin: auto;    
+
+.media-icons a:not(:last-child) {
+  margin-right: 200px;
 }
-.media-icons a{
-    position: relative;
-    color: #111;
-    font-size: 30px;
+
+.media-icons a:hover {
+  transform: scale(1.5);
 }
-.media-icons a:not(:last-child){
-    margin-right: 200px;
+
+.header .navigation a:not(:last-child) {
+  margin-right: 30px;
 }
 
-.media-icons a:hover{
-    transform: scale(1.5);
+.media-icons {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
 }
+
+/* .media-icons {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
+    padding: 12px 25px 35px 0;
+  } */
```

Comment:
Please do not leave unused code as comments on your code. Unused code as comments makes your code look messy.